JHUGenMELAv2/MELA/interface/TMCFM.hh

```diff
@@ -238,6 +238,17 @@ extern "C" {
     double gh2w1_prime6[2]; double gh2w2_prime6[2]; double gh2w3_prime6[2]; double gh2w4_prime6[2]; // q1**4 - q2**4
     double gh2w1_prime7[2]; double gh2w2_prime7[2]; double gh2w3_prime7[2]; double gh2w4_prime7[2]; // |q1**2| * |q2**2|
 
+/*
```

Comment:
Also, the order of variables should be bool, ten int, then double precision, then complex.